lmnet/lmnet/datasets/dataset_iterator.py

```diff
@@ -1,6 +1,5 @@
 from multiprocessing import Pool
```

Comment:
Should this have the copyright label as well?

Reply:
@nlpng 
Oh, thanks. I will add copyright.

Reply:
changed https://github.com/blue-oil/blueoil/pull/166/commits/667d13857325ceccfa463d49024bcfb9d0556ffe